presto-product-tests/conf/docker/files/run-tempto.sh

```diff
@@ -15,6 +15,7 @@ if ! test -z ${TEMPTO_EXTRA_CONFIG_FILE:-}; then
 fi
 
 java \
+  `#-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5007` \
```

Comment:
I don't run with agentlib always -- https://github.com/prestodb/presto/pull/9472 -- so this is commented out. 
This fancy `` `#...` `` is just the way you put inline comments in bash.

Reply:
Maybe `` `#echo -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5007` `` than?

Reply:
how is this better?

Reply:
I tried to run `` echo `-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5007` ``, and it fails with `bash: -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5007: command not found`, as it tries to execute the `-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5007` part

Reply:
I thought the idea is that you can simply remove `#` to enable debug port

Reply:
The idea is that you remove `` `# `` and `` ` `` to enable debug port. I think what you propose (`` `#echo ...`` -- in your comment above you had this mis-typed) would work and indeed one would need to remove just `#`. But then, it would be less obvious, what to do, so it might not be a time saver after all.

Reply:
Don't bother then, i misunderstood. I though the intention was to simply remove a single character.